SwiftI18n/Classes/Main/I18nManager.swift

```diff
@@ -37,8 +38,8 @@ public class I18nManager {
     /// Setting `fallbackLanguage` should be done with consideration of these potential overrides
     public var fallbackLanguage: String?
     
-    public var localizationPerformingBlock: (_ key: String, _ language: String)->(String) = { (key, language) in
-        return NSLocalizedString(key, tableName: language, comment: "")
+    public var localizationPerformingBlock: (_ key: String, _ language: String, _ bundle: Bundle) -> (String) = { (key, language, bundle) in
```

Comment:
Can we also default it to main?

Reply:
```suggestion
    public func localizationPerformingBlock(_ key: String, _ language: String, _ bundle: Bundle = .main) -> (String) {
        NSLocalizedString(key, tableName: language, bundle: bundle, comment: "")
    }
```

If we add a default parameter and change this block to a `func`, would it break the public interface? I don't think so